scipy/spatial/ckdtree.pyx

```diff
@@ -1064,17 +1110,37 @@ cdef public class cKDTree [object ckdtree, type ckdtree_type]:
         p : float, optional 
             1<=p<=infinity, default 2.0
             Which Minkowski p-norm to use
+        self_weights : array-like or None, optional
+            weight of each data point in self. 
+            If None, the pair-counting is unweighted (weighted by 1) for 'self'.
+            Default: None
+        other_weights : array-like or None, optional
+            weight of each data point in other. 
+            If None, the pair-counting is unweighted (weighted by 1) for 'other'.
+            Default: None
+        cumulative : boolean
+            Whether the returned counts are cumulative. 
 
         Returns
         -------
-        result : int or 1-D array of ints
-            The number of pairs.
+        result : scalar or 1-D array
+            The number of pairs. For unweighted counts, the result is integer.
+            For weighted counts, the result is float.
+            If cumulative is False, `result[i]` contains the counts with
+            `r[i-1] or -inf < R <= r[i]`
```

Comment:
use double backticks here and in the line above, then it will be typeset as code (single backticks are references)